### ISSUE TITLE
Incorrectly flagged for insufficient color contrast ratio

### ISSUE THREAD

**Issue by beachdbishop** [pa11y/pa11y-ci#238]
Scanning a site in my dev env with pa11y-ci 3.1.0 on Windows 10.

> Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.12:1

Isn't 4.12:1 higher/better than 4.5:1?

Double-checked the contrast w/ Polypane just to make sure:
![image](https://github.com/pa11y/pa11y-ci/assets/94011380/6f280c23-7d3e-413f-8a9c-6e85055a6e7e)

How can I resolve this?

**Comment by kussmaul** [comment]
Isn't 4.12 less than 4.5? :-)

**Comment by hollsk** [comment]
Hey @beachdbishop! Unfortunately, as @kussmaul says, 4.12 is lower than 4.5 😅 

You can plug your hex codes into the [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/) and play with the tools there to see how changes are reflected in the ratio.

**Comment by beachdbishop** [comment]
> Isn't 4.12 less than 4.5? :-)

OMG. Thank you, fellow human. Apparently, the coffee hasn't hit.
